protected/humhub/modules/space/controllers/SpaceController.php

```diff
@@ -43,7 +45,8 @@ public function behaviors()
         return [
             'acl' => [
                 'class' => AccessControl::class,
-                'guestAllowedActions' => ['index', 'home', 'stream']
+                'guestAllowedActions' => ['index', 'home', 'stream'],
+                'guestAllowedActions' => ['index', 'about', 'stream'],
```

Comment:
Duplicate?

Reply:
I was forced to add `'stream'` because it was displaying weirdly and once I added this it worked just fine :thinking:

Reply:
`'guestAllowedActions' => ['index', 'home', 'about', 'stream'],`

Reply:
Should be resolved now